foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -117,7 +120,11 @@ public static ConcurrentCompositeConfiguration createLocalConfig() {
 
     LOGGER.info("create local config:");
     for (ConfigModel configModel : loader.getConfigModels()) {
-      LOGGER.info(" {}.", configModel.getUrl());
+      DynamicBooleanProperty isPrintUrl = DynamicPropertyFactory.getInstance()
```

Comment:
Do not write this in loop. Maybe append all info to a StringBuilder and log it in one line.

Reply:
add toString